test/test.py

```diff
@@ -485,6 +475,30 @@ def to_json(fqid):
 
         self.assertEqual({}, errors)
 
+    def test_sequencing_process_paired_end(self):
```

Comment:
Can you make the code move a separate commit?

Reply:
Could you elaborate the change from `'prod'` to `''`?

Reply:
Yes, i'll make the code move a separate commit. I moved the function to group tests that don't use _test_bundle() below _test_bundle().

Reply:
Regarding changing 'prod' to '': deployment = "prod" works for _load_bundle() but not if it needs to download the bundle because dss_client() needs it to be empty for prod.

Longer version: The "deployment" var is passed to _load_bundle().
In _load_bundle() the value of the var (or string "prod" if empty) is copied to a var "canning_directory".
If _load_bundle() needs ot download the bundle, dss_client() is called passing the "deployment" var.
dss_client() uses the "deployment" param to build the swagger url, requiring "deployment" to be empty if production is desired (otherwise you'd get https://dss.prod.data.humancellatlas.org...)

Reply:
I shouldn't have to modify the call to _load_bundle(). The same arguments should both work for initially downloading a bundle and canning it as well as loading an already canned bundle. Is there a bug lurking that you change is merely working around?